support/cas-server-support-authy/src/main/java/org/apereo/cas/adaptors/authy/config/AuthyConfiguration.java

```diff
@@ -38,6 +39,8 @@
  * @since 5.0.0
  */
 @Configuration("authyConfiguration")
+@ConditionalOnMultifactorTrustedDevicesEnabled(prefix = "cas.authn.mfa.authy")
```

Comment:
This is not correct, and should be removed. Trusted-devices feature is separate from MFA, and requiring both is a breaking change in behavior.